server.go

```diff
@@ -1443,6 +1518,45 @@ func (sp *serverPeer) OnGetAddr(p *peer.Peer, msg *wire.MsgGetAddr) {
 	sp.pushAddrMsg(addrCache)
 }
 
+// OnGetAddrV2 is invoked when a peer receives a getaddrv2 wire message and is
+// used to provide the peer with known addresses from the address manager.
+func (sp *serverPeer) OnGetAddrV2(p *peer.Peer, msg *wire.MsgGetAddrV2) {
```

Comment:
As discussed elsewhere, I don't think a separate `getaddrv2` is necessary since it is identical to `getaddr` and the protocol version dictates whether to respond with an `addr` or `addrv2` message.